cinema/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from .views import retrieve_cinema, get_movies
+
+urlpatterns = [
+    path("movies/", get_movies, name="movies-list"),
+    path("movies/<int:pk>/", retrieve_cinema, name="movie-detail"),

```

Comment:
```suggestion
    path("movies/<int:pk>/", retrieve_movie, name="movie-detail"),

```

Reply:
btw, get_movies is a bad name, because you are not only getting movies there. Same with retrieve